//  Copyright (c) 2017, John Manferdelli, All rights reserved.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
syntax = "proto2";
package tao_support;

message SavedProgramKeys {
	optional bytes sealed_signing_key = 1;
	optional bytes sealed_crypting_key = 2;
	repeated bytes signer_cert_chain = 3;
}
